docker-compose.yml

```
@@ -3,15 +3,15 @@ version: "3.7"
 services:
   mojaloop-testing-toolkit:
     image: mojaloop/ml-testing-toolkit:v14.1.0
-    #image: mojaloop-testing-toolkit:local
+    #image: mojaloop/ml-testing-toolkit:local
     #build:
     #  context: .
     #  target: builder
     volumes:
       - "./spec_files:/opt/mojaloop-testing-toolkit/spec_files"
```

```suggestion
      - "./spec_files:/opt/appspec_files"
```

Note: please check for any other places where the directory will need to be updated